mail/contact_me.php

```diff
@@ -16,11 +16,11 @@
 $message = strip_tags(htmlspecialchars($_POST['message']));
 	
 // Create the email and send the message
-$to = 'yourname@yourdomain.com'; // Add your email address inbetween the '' replacing yourname@yourdomain.com - This is where the form will send a message to.
-$email_subject = "Website Contact Form:  $name";
-$email_body = "You have received a new message from your website contact form.\n\n"."Here are the details:\n\nName: $name\n\nEmail: $email_address\n\nPhone: $phone\n\nMessage:\n$message";
-$headers = "From: noreply@yourdomain.com\n"; // This is the email address the generated message will be from. We recommend using something like noreply@yourdomain.com.
-$headers .= "Reply-To: $email_address";	
+$to = 'hello@codeforpoznan.pl'; // Add your email address inbetween the '' replacing hello@codeforpoznan.pl - This is where the form will send a message to.
+$email_subject = "Formularz kontaktowy Code For Poznan:  $name";
+$email_body = "Otrzymałeś nową wiadomość z formularza kontaktowego codeforpoznan.pl.\n\n"."Oto szczegóły:\n\nImię: $name\n\nEmail: $email_address\n\nTelefon: $phone\n\nMessage:\n$message";
+$headers = "From: noreply@codeforpoznan.pl\n"; // To jest adres e-mail, z którego została wygenerowana wiadomość. W celu kontaktu zalecamy użyć hello@codeforpoznan.pl";
```

Comment:
We don't have to translate comments. ;-)

Reply:
You're right ;)